File: Li223/Sarcasm.cs
static void Main(string[] _)
{
  var text = "This does not align with our core values".ToArray();

  for (var i = 1; i < text.Length; i++)
  {
    var cnt = 1;
    if (char.IsWhiteSpace(text[i])) cnt++;
    if (char.IsLower(text[i - cnt])) text[i] = char.ToUpperCase(text[i]);
    else text[i] = char.ToLowerCase(text[i]);
  }
  Console.WriteLine(string.Join("", text));
  Console.ReadLine();
}

File: ElectronApplications/Sarcasm.js
const specialSymbols = [" ", ".", ",", "?", "!", "'"];
var text = "this does not align with our core values".split("");

for (let i = 0, j = 0; i < text.length; i++) {
    if(specialSymbols.includes(text[i])) {
        continue;
    }
    
    if(j % 2 == 0)
        text[i] = text[i].toLowerCase();
    else
        text[i] = text[i].toUpperCase();
    
    j++;
}

console.log(text.join(""));